drivers/dac/ad9152/ad9152.h

```diff
@@ -56,7 +57,7 @@
 #define REG_PWRCNTRL0                            0x011 /* Power Control Reg 1 */
 #define REG_TXENMASK1                            0x012 /* TXenable masks */
 #define REG_PWRCNTRL3                            0x013 /* Power control register 3 */
-#define REG_COARSE_GROUP_DLY                     0x014 /* Coarse Group Delay Adjustment */
+#define REG_PWRCNTRL1                            0x014 /* Power control register 1 */
```

Comment:
i'm not sure what happened here, first contribution to this seems to be from 10yrs ago, maybe it was based on a preliminary datasheet, because some registers got completely moved around